src/web-vitals-autoinstrumentation.ts

```diff
@@ -448,21 +464,27 @@ export class WebVitalsInstrumentation extends InstrumentationAbstract {
 
     const { name, attribution } = ttfb;
     const {
-      waitingTime,
-      dnsTime,
-      connectionTime,
-      requestTime,
+      cacheDuration,
+      connectionDuration,
+      dnsDuration,
+      requestDuration,
+      waitingDuration,
     }: TTFBAttribution = attribution;
     const attrPrefix = this.getAttrPrefix(name);
 
     const span = this.tracer.startSpan(name);
 
     span.setAttributes({
       ...this.getSharedAttributes(ttfb),
-      [`${attrPrefix}.waiting_time`]: waitingTime,
-      [`${attrPrefix}.dns_time`]: dnsTime,
-      [`${attrPrefix}.connection_time`]: connectionTime,
-      [`${attrPrefix}.request_time`]: requestTime,
+      [`${attrPrefix}.waiting_duration`]: waitingDuration,
+      [`${attrPrefix}.dns_duration`]: dnsDuration,
+      [`${attrPrefix}.connection_duration`]: connectionDuration,
+      [`${attrPrefix}.request_duration`]: requestDuration,
+      [`${attrPrefix}.cache_duration`]: cacheDuration,
+      // [`${attrPrefix}.waiting_time`]: waitingDuration, // TODO:  Deprecate instead?
```

Comment:
I do like duration better than time in this case, BUT in order to not break any existing telemetry can we double send these values for a version while we make sure the dashboard is up to date with these values?